scalafmt-tests/src/test/resources/dotty/Enum.stat

```diff
@@ -1,3 +1,4 @@
+runner.dialect = Scala3
```

Comment:
Dialect is inferred based on the fact that it's in the dotty directory. Similar to how it was done for ScalaJS. No need to set it here.

Reply:
Nice, removed.